.github/workflows/gradle.yml

```diff
@@ -685,6 +685,167 @@ jobs:
           label: ${{ needs.start-kube-acceptance-test-runner.outputs.label }}
           ec2-instance-id: ${{ needs.start-kube-acceptance-test-runner.outputs.ec2-instance-id }}
 
+
+  ## Kube Acceptance Tests
+  # Docker acceptance tests run as part of the build job.
+  # In case of self-hosted EC2 errors, remove this block.
+  start-helm-acceptance-test-runner:
```

Comment:
nice! I'm guessing we will remove the current kustomize test in a follow up PR?

Reply:
I think we can, but let's discuss it on ours todays sync-up